_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1057,6 +1163,25 @@ locals {
   # Both buckets will be created in the logs account by account-baseline-root
   config_s3_bucket_name     = "acme-config-bucket-logs"
   cloudtrail_s3_bucket_name = "acme-cloudtrail-logs"
+
+  # The following locals are used for constructing multi region provider configurations for the underlying module.
+  # A list of all AWS regions
+  all_aws_regions = [
+    "af-south-1",
+    "ap-east-1",
+    "ap-northeast-1",
+    # ...,
```

Comment:
Why `...`? That means this code can't be copy/pasted. Any reason not to have the full list here?

Reply:
Addressed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/647/commits/1c37131de4f954365d95d4c80c2c8349a6e66ee8.